lib/danger-packwerk/danger_packwerk.rb

```diff
@@ -67,9 +70,13 @@ def check(
       # trigger the warning message, which is good, since we only want to trigger on new code.
       github.dismiss_out_of_range_messages
 
+      # TODO: write explanation here
```

Comment:
I think it's fine to just put the explanation once in the `GitFilesystem` class.